java/jetty/testdata/java11_debug.yaml

```diff
@@ -12,6 +12,10 @@ commandTests:
     command: "/usr/bin/java"
     args: ["-version"]
     expectedError: ['openjdk version "11\.*']
+  - name: javac
+    command: "/usr/lib/jvm/java-11-openjdk-amd64/bin/javac"
```

Comment:
Could you also add tests for non-debug images (`java8/11.yaml`) to test the `javac` does not exist? I think this can be done using `fileExistenceTests` with `shouldExist: false` like below.